src/analysis/retail/warrior/protection/modules/talents/BurstOfPower.tsx

```diff
@@ -0,0 +1,25 @@
+import SPELLS from 'common/SPELLS';
+import TALENTS from 'common/TALENTS/warrior';
+import Analyzer, { Options } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+import SpellUsable from 'parser/shared/modules/SpellUsable';
+
+export default class BurstOfPower extends Analyzer.withDependencies({
+  spellUsable: SpellUsable,
+}) {
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasTalent(TALENTS.BURST_OF_POWER_TALENT);
+
+    this.addEventListener(Events.applybuff.spell(SPELLS.BURST_OF_POWER_BUFF), this.resetShieldSlam);
+
+    this.addEventListener(
+      Events.removebuffstack.spell(SPELLS.BURST_OF_POWER_BUFF),
+      this.resetShieldSlam,
+    );
```

Comment:
should this include `removebuff`? aka is it "Buff application resets cooldown, then the next cast has no cooldown." or "each cast with the buff active does not trigger the cooldown, but there is no reset on buff apply"? (or some hybrid?)

this is one of those cases where the talent tooltip doesn't tell the full story and it is worth documenting the expected behavior here in brief in case it changes later